lib/generator/data/enum_definition.dart

```diff
@@ -29,11 +29,18 @@ class EnumDefinition extends Definition with DataPrinter {
 /// Enum value name
 class EnumValue extends Name with DataPrinter {
   /// Instantiate a enum value name definition.
-  EnumValue({@required String name}) : super(name: name);
+  EnumValue({
```

Comment:
I think it is better to move towards the same approach as in classes.

We will get 
`EnumDefinition` -> for enum
`EnumValueDefinition` -> for name and annotations
and rename `EnumValue` => `EnumValueName`